Sources/xcbeautify/Xcbeautify.swift

```diff
@@ -66,8 +66,9 @@ struct Xcbeautify: ParsableCommand {
         }
 
         if !report.isEmpty {
-            let outputPath = URL(fileURLWithPath: FileManager.default.currentDirectoryPath)
-                .appendingPathComponent(reportPath)
+            let outputPath = URL(fileURLWithPath: reportPath,
+                                 relativeTo: URL(fileURLWithPath: FileManager.default.currentDirectoryPath))
```

Comment:
According to my tests, this handles both relative and absolute `reportPath` values.